internal/config/defaults.go

```diff
@@ -12,7 +12,8 @@ var Defaults = map[string]any{
 	"endpoint.max_page":  5,
 	"endpoint.per_page":  100,
 
-	"view.height": 40,
+	"view.height":   40,
+	"view.log_path": path.Join(StateDir(), "gh-not-debug.log"),
```

Comment:
I would make the log file name simpler, given that `StateDir()` already contains `gh-not`.

```suggestion
	"view.log_path": path.Join(StateDir(), "debug.log"),
```